packages/devextreme/js/__internal/ui/m_tag_box.ts

```diff
@@ -738,8 +741,13 @@ const TagBox = (SelectBox as any).inherit({
       selectedItems: this.option('selectedItems'),
     };
 
+    let tagData = '';
```

Comment:
1. U don't need to create a variable here, u can just create it and assign the value in a single row below
2. When I read `tagData` name  I image that it's a data object. I think that `tagText` name is more suitable here.